src/ripple/basics/BasicConfig.h

```diff
@@ -88,7 +88,6 @@ class Section
             lines_.emplace_back (std::move (value));
         else
         {
-            assert (lines_.size () == 1);
             lines_[0] = std::move (value);
```

Comment:
This is fine. Since you're here, can you remove the, now superfluous curly brackets around the `else` and the unneeded and confusing `else` further down, on line 106 where a simple  `if (lines_.size () > 1)` should be enough.